tubular/mapping.py

```diff
@@ -217,7 +217,70 @@ def transform(
         return X
 
 
-class CrossColumnMappingTransformer(BaseMappingTransformer):
+class BaseCrossColumnMappingTransformer(BaseMappingTransformer):
```

Comment:
P - good shout on pulling this out into a base class.  A case could be made for creating a new sub module for cross column transformers but for now I think this is fine.  (and these test updates hopefully make such a refactor very straightforward in future)

Reply:
Thanks David :) agree could be worth splitting in future